src/QMCWaveFunctions/LCAO/MultiQuinticSpline1D.h

```diff
@@ -47,11 +51,20 @@ struct LogGridLight
     double log_ratio  = std::log(ratio);
     double dlog_ratio = log_ratio / static_cast<double>(n - 1);
     OneOverLogDelta   = 1.0 / dlog_ratio;
+    LogDelta          = dlog_ratio;
     r_values.resize(n);
     for (int i = 0; i < n; i++)
       r_values[i] = ri * std::exp(i * dlog_ratio);
   }
 
+  PRAGMA_OFFLOAD("omp declare target")
+  static inline T getCL(T r, int& loc, double OneOverLogDelta_, T lower_bound_, double dlog_ratio_)
```

Comment:
Avoid using `_` it violates coding convention. Upper case is also reserved for type names. So `OneOverLogDelta_` is bad.
`OneOverLogDelta_` => `one_over_log_delta`
`lower_bound_` => `lower_bound`
`dlog_ratio_` => `log_delta`